### ISSUE TITLE
Configurable policies conflicting with the Epsilon functionality

### ISSUE THREAD

**Issue by tkchouaki** [eqasim-org/eqasim-java#280]
The configurable policies introduced in #254 use a prefix on the utility estimator name to identify which ones to override with the policy estimator. The epsilon module does the same thing and the presence of both prefixes causes epsilon estimators to not be bound.

**Comment by sebhoerl** [comment]
Yes, already noticed that for Munich. You'll have to do a little quick fix for now:
https://github.com/eqasim-org/eqasim-java/commit/f4262bbd45bd6c9e5db88b1d5508412396216f10

**Comment by sebhoerl** [comment]
ah no, not sure if this is actually the proper fix
